exasol/driver/websocket/_types.py

```diff
@@ -0,0 +1,96 @@
+"""
+This module provides `PEP-249`_ a DBAPI compliant types and type conversion definitions.
+(see also `PEP-249-types`_)
+
+.. _PEP-249-types: https://peps.python.org/pep-0249/#type-objects-and-constructors
+"""
+from datetime import (
+    date,
+    datetime,
+    time,
+)
+from enum import (
+    IntFlag,
+    auto,
+)
+from time import localtime
+
+
+def Date(year: int, month: int, day: int) -> date:  # pylint: disable=C0103
```

Comment:
Why use an upper-case name for a function?
And why do you need this function at all?